templates/kubernetes/compare.html

```diff
@@ -372,6 +372,7 @@ <h3 class="u-sv2">Kubernetes resources</h3>
   </div>
   <div class="row p-divider">
     <div class="col-4 p-divider__block">
+      <!-- TODO: add missing img and title classes -->
```

Comment:
Missing image? I see a datasheet one below.

Reply:
Nevermind, this was a TODO comment for myself to know which classnames are missing, I forgot to remove it.

Reply:
```suggestion
```